Source/Fuse.ImageTools/Tests/ImageTools.Test.uno

```diff
@@ -0,0 +1,48 @@
+using Uno;
+using Uno.Collections;
+using Uno.Testing;
+using Fuse;
+using FuseTest;
+
+namespace Fuse.ImageTools.Test
+{
+	public class ImageToolsTest : TestBase
+	{
+		private Object _image = null;
+
+		[Test]
+		[Ignore("Only supported on dotnet, Android or iOS", "native")]
```

Comment:
Now the comment and condition are a mismatch.... "native" means "CMake" or "MSVC" on macOS and Windows respectively.

I think what you want is `[Ignore("Only supported on mobile platforms", "MOBILE")]`